homeassistant/components/sma/sensor.py

```diff
@@ -84,6 +84,9 @@ def __init__(
     @property
     def name(self) -> str:
         """Return the name of the sensor."""
+        if self._attr_device_info is not None:
+            return f"{self._attr_device_info.get('name')} {self._sensor.name}"
```

Comment:
`get()` van return `None`, which can result in `None <Sensor name>`?

Reply:
It should always be set, but I have update the code to prefix the name with "SMA" if `_attr_device_info`  or `_attr_device_info["name"]` is not set